runtime/src/main/java/org/corfudb/runtime/CheckpointWriter.java

```diff
@@ -126,8 +127,18 @@ public CheckpointWriter(CorfuRuntime rt, UUID streamId, String author, T map) {
      */
```

Comment:
I know this is not relevant to your change, but can we update the return description for this method?

Reply:
Done.